CruCentralCoast/Login/ForgotPasswordVC.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ForgotPasswordVC.swift
+//  CruCentralCoast
+//
+//  Created by Michael Cantrell on 9/12/18.
+//  Copyright © 2018 Landon Gerrits. All rights reserved.
+//
+
+import UIKit
+import FirebaseAuth
+
+
+class ForgotPasswordVC: UIViewController {
+    @IBOutlet weak var emailTextField: CruTextField!
+    var email: String? { didSet { self.emailTextField?.setText(self.email) } }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.emailTextField.setText(self.email)
+    }
+    
+    @IBAction func sendEmailButtonPressed(_ sender: Any) {
```

Comment:
Shouldn't this be named "resetPasswordButtonPressed"?

Reply:
Pro Tip: You don't need the "_ sender: Any" parameter for an IBAction. Just in case you were unaware. It doesn't matter to me if you leave it or remove it.